src/core/sys/posix/netinet/tcp.d

```diff
@@ -26,23 +26,4 @@ extern (C):
 TCP_NODELAY
 */
 
-version( linux )
-{
-    enum TCP_NODELAY = 1;
-}
-else version( OSX )
-{
-    enum TCP_NODELAY = 1;
-}
-else version( FreeBSD )
-{
-    enum TCP_NODELAY = 1;
-}
-else version( Solaris )
-{
-    enum TCP_NODELAY = 1;
-}
-else version( Android )
-{
-    enum TCP_NODELAY = 1;
-}
+enum TCP_NODELAY = 1;
```

Comment:
[Must be defined in POSIX standard, but value is not specified](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/netinet_tcp.h.html).